exercises/practice/protein-translation/.meta/example.rkt

```diff
@@ -0,0 +1,24 @@
+#lang racket
+
+(provide proteins)
+
+(define (translate protein)
```

Comment:
I'd call this `unsafe-translate` to highlight that this function is unsafe to use and can potentially raise an exception at runtime. We don't need to update the `proteins` name though.